src/main/java/racingcar/rule/FixedRacingRule.java

```diff
@@ -2,14 +2,24 @@
 
 public class FixedRacingRule implements RacingRule{
```

Comment:
```suggestion
public class FixedRacingRule implements RacingRule {
```
클래스명과 중괄호 사이에 띄어쓰기는 꼭 넣어주세요
코딩 컨벤션입니다

Reply:
네 변경하였습니다